Gems/Atom/RHI/WebGPU/Code/Source/RHI/BindGroup.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <Atom/RHI/DeviceObject.h>
+#include <Atom/RHI/DeviceBuffer.h>
+#include <Atom/RHI/DeviceBufferView.h>
+#include <Atom/RHI/DeviceImage.h>
+#include <Atom/RHI/DeviceImageView.h>
+#include <Atom/RHI.Reflect/SamplerState.h>
+#include <AzCore/std/containers/span.h>
+#include <AzCore/Memory/PoolAllocator.h>
+#include <RHI/Buffer.h>
+
+namespace AZ::RHI
+{
+    class ResourceView;
+}
+
+namespace AZ::WebGPU
+{
+    class BufferView;
+    class BindGroupLayout;
+    class Device;
+    class ImageView;
+    class Sampler;
+
+    //! Encapsulate a WebGPU BindGroup.
+    class BindGroup final
+        : public RHI::DeviceObject
+    {
+        using Base = RHI::DeviceObject;
```

Comment:
do you need it outside like in the DescriptorPool ? If not i would use __super instead